tests/constraints/test_float_constraints.py

```diff
@@ -83,7 +88,7 @@ def test_handle_constrained_float_handles_multiple_of_with_lt(val1: float, val2:
 )
 def test_handle_constrained_float_handles_multiple_of_with_le(val1: float, val2: float) -> None:
     multiple_of, max_value = sorted([val1, val2])
-    if multiple_of == 0 or max_value > multiple_of:
```

Comment:
Checks like `max_value > multiple_of` make no sens since we always do `sorted` first + if range is unbounded on either edge we can always find a multiply of multple_of